config.json

```diff
@@ -163,6 +163,52 @@
         "title": "Add praise for the actions of your colleagues.",
         "color": "#00c7e6"
       }
+    },
+    "Santa": {
+      "Sleepy Santa": {
+        "title": "Sleepy Santa",
+        "color": "#f44335"
+      },
+      "Happy Snowman": {
+        "title": "Happy Snowman",
+        "color": "#00bcd4"
+      },
+      "Sassy Scrooge": {
+        "title": "Sassy Scrooge",
+        "color": "#ff5630"
+      },
+      "Busy Elf": {
+        "title": "Busy Elf",
+        "color": "#4baf4f"
+      },
+      "Energized Reindeer": {
+        "title": "Energized Reindeer",
+        "color": "#ffc400"
+      },
+      "Good - presents": {
+        "title": "What were our gifts?",
+        "color": "#00bcd4"
+      },
+      "Support - Elves": {
+        "title": "Who were our helpers?",
+        "color": "#4baf4f"
+      },
+      "Bad- Grinch": {
```

Comment:
Are those the actual JSON keys from the MR response? Otherwise it won't work. Asking because the `-` is stuck to the previous word here, and not on the other ones :)

Reply:
Yeah, I exported the JSON and used the same ones - I tested it locally :)

Reply:
Ok! Weird then ^^